docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
+------------------
+
+.. versionadded:: 6.0.0
+
+The OME-TIFF specification supports multi-resolution images or pyramidal
+images where individual planes are stored at different levels of resolution.
+The intermediate downsampled image planes are called pyramidal levels,
+sub-resolution image planes or sub-resolutions.
+
+Supported resolutions
+^^^^^^^^^^^^^^^^^^^^^
+
+OME-TIFF planes can be reduced alongside the X and Y dimensions. Each pyramidal
+level must be a downsampling of the full-resolution plane in the X and Y
+dimensions and the resolution should stay unchanged in the other dimensions.
+The downsampling factor should be an integer value, identical alongside the X
+and the Y dimensions and stay the same between each consecutive pyramidal
+level.
+
+The following table below shows two examples of pyramidal level dimensions
+using typical downsampling factors:
+
+.. list-table::
+  :header-rows: 1
+
+  -  *
+     * Example 1
+     * Example 2
+
+  -  * Downsampling factor
+     * 3
+     * 4
+
+  -  * Level 0 (full-resolution)
+     * 9234 × 6075 × 1 × 1 × 10
+     * 38912 × 25600 × 1 × 3 × 1
+
+  -  * Level 1
+     * 3078 × 2025 × 1 × 1 × 10
+     * 9728 × 6400 × 1 × 3 × 1
+
+  -  * Level 2
+     * 1026 × 675 × 1 × 1 × 10
+     * 2432 × 1600 × 1 × 3 × 1
+
+  -  * Level 3
+     * 342 × 225 × 1 × 1 × 10
+     * 608 × 400 × 1 × 3 × 1
+
+  -  * Level 4
+     * 114 × 74 × 1 × 1 × 10
+     * 152 × 100 × 1 × 3 × 1
+
+  -  * Level 5
+     * 38 × 25 × 1 × 1 × 10
+     * 38 × 25 × 1 × 3 × 1
+
+
+Storage
+^^^^^^^
+
+Full-resolution image planes must reamin stored as described above using a valid TIFF IFD and referenced from the OME-XML metadata using the
+:ref:`TiffData <tiffdata>` element.
+
+Each sub-resolution must be stored in a valid IFD of the same TIFF file as the full-resolution image plane. Additionally:
+
+- the offsets of all sub-resolutions IFDs must be referenced from the IFD of
+  the full-resolution plane using the `SubIFDs` TIFF extension tag. The list 
+  of sub-resolution offsets must be ordered by plane size from largest to
+  smallest,
+- the IFD offsets of pyramidal levels must not be referenced in the primary
+  chain of IFDs derived from the first IFD of the TIFF file,
+- the NewSubFileType TIFF tag for each pyramidal level should be set to 1
```

Comment:
Worth adding the tag number (254)?

Reply:
Also, should `NewSubFileType` be recommended to be set to 0x0 (full-resolution) for all primary IFDs?  We already do this in OME Files for completeness.

Reply:
No objection to recommending setting `NewSubFileType` of the primary `SubIFDs` either as a `should` or a `may`. Similarly to https://github.com/ome/ome-model/pull/98#discussion_r249953035, this probably would apply to any OME-TIFF independent of sub-resolutions and I would review this in a separate PR.

I was slightly confused by the page bit and especially when the classification of multi-page images apply. Looking at https://www.loc.gov/preservation/digital/formats/fdd/fdd000237.shtml as an example, is it correct that:
- an OME-TIFF with a single plane with multiple sub-resolutions would set `NewSubFileType` 0 on the primary IFD (full-resolution, single page) and 1 on each SubIFD (reduced resolution)
- an OME-TIFF with multiple planes with multiple sub-resolutions would  set `NewSubFileType` 2 on each primary IFD (full-resolution, multiple page) and 1 on each SubIFD (reduced resolution)
- an OME-TIFF with multiple plane with no sub-resolutions would set `NewSubFileType` 2 on each IFD (full-resolution, multiple page)